modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -209,6 +209,33 @@ mirror:
     targetCatalog: my-namespace/my-operator-catalog
 ----
 
+// New example; multi-arch; all versions
+[discrete]
+[id="oc-mirror-image-set-examples-multi-arch_{context}"]
+== Use case: Including all versions of OpenShift Container Platform from a minimum to the latest
+
+The following `ImageSetConfiguration` file sets the `architectures` field to `multi` to include `multi` as the platform architecture which mirrors a multi-arch release image.
+
+Previously, oc-mirror users are able to mirror x86_64 release images when no platform architecture is specified in imageSetConfig. They could also mirror other architectures individually by adding other platform architectures in the imageSetConfig.
+
+With this feature, users can specify multi as the platform architecture which mirrors a multi-arch release image.
+
+
+.Example `ImageSetConfiguration` file
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+mirror:
+  platform:
+architectures:
+   - "multi" # Valid values are: amd64(default), arm64, ppc64le, s390x and multi.
+channels:
+   - name: stable-4.9
+     minVersion: 4.9.13
+     maxVersion: 4.9.26
+----
```

Comment:
@subhtk Above example does not work to mirror multi arch releases. Please help update the example as below, thanks !!
apiVersion: mirror.openshift.io/v1alpha2
kind: ImageSetConfiguration
mirror:
  platform:
architectures:
   - "multi" # Valid values are: amd64(default), arm64, ppc64le, s390x and multi.
channels:
   - name: stable-4.13
     minVersion: 4.13.4
     maxVersion: 4.13.6

Reply:
Ack.